### ISSUE TITLE
Stuck on "Mounting USB/SD device

### ISSUE THREAD

**Issue by petitmulot** [FIX94/Nintendont#664]
I'm trying to use Nintendont v5.486 on my Wii
I use a transcend 16go SD card for the apps and only have BootMii and Nintendon't on it.
Created "apps" folder and created Nintendon't in it.
transfered icon + meta + loader(renamed "boot")

I use a Lexar 128go USB3.0 flash drive for my games
formatted the USB to FAT32, 32k with macrorit software
I made a folder on the root of it and called games and placed some games there.

Here's the tricky part:
I'm able to play games if the EmulateMemoryCard option is set to OFF, but it won't boot if I change it to ON !

If I try to launch a game in nintendont it get stuck on the loading message or on "Mounting USB/SD device..." after doing:
"Loading patched kernel... 3
ES_Init... Done!
Initing soirage devices... Done!"

I've been working on it for HOURS now !
Sorry for my bad english, english isnt my first language
Please help me !

**Comment by carnage702** [comment]
wii or wiiu?

also does your usb device even get to be created the saves folder? it seems nintendont cant write to your hdd, you can try using another formatter program not sure if macorite does it  correctly.

**Comment by petitmulot** [comment]
On Wii. I've created a SAVE folder and it works with the first game I try, but I got the same issue for all other games...

**Comment by carnage702** [comment]
probably your hdd is not formated correctly or it has some issues as nintendont cannot creat memory cards on it, but take a picture of your hdd root,nintendont settings and your save folder.

**Comment by BeenaSon** [comment]
I am having the exact same issue!  When memory card emulation is disabled all of my games work flawlessly.  When it's enabled then only a couple of games work (like Animal Crossing and Super Smash Bros. Melee) for whatever reason...  I reformatted my USB drive multiple times and am using a 32 GB FAT32 flash drive with 32k clusters so I'm not sure why it's not working...?  I even put the USB drive back in my computer and see that saves are being created in the saves folder by Nintendont for each game I attempt to run.  However, every file is 0 KB in size with the exception of Animal Crossing which is working flawlessly for whatever reason.  Any help would be greatly appreciated!

**Comment by carnage702** [comment]
0kb files means they are bad save files, what settings do you have set to on on nintendont? can take a picture?

**Comment by carnage702** [comment]
> 
> 
> I am having the exact same issue! When memory card emulation is disabled all of my games work flawlessly. When it's enabled then only a couple of games work (like Animal Crossing and Super Smash Bros. Melee) for whatever reason... I reformatted my USB drive multiple times and am using a 32 GB FAT32 flash drive with 32k clusters so I'm not sure why it's not working...? I even put the USB drive back in my computer and see that saves are being created in the saves folder by Nintendont for each game I attempt to run. However, every file is 0 KB in size with the exception of Animal Crossing which is working flawlessly for whatever reason. Any help would be greatly appreciated!

0kb files means they are bad save files, what settings do you have set to on on nintendont? can take a picture?

**Comment by BeenaSon** [comment]
Well I replied via email, but it doesn't look like it made it to the site for some reason so here it is...

"Thanks so much for the super quick response!  I took a picture of my settings and attached it to this email per your request.  I also saw in the other post you requested a picture of the OP's USB HDD Root and save folder so I included screenshots of those too.  Please let me know if you need anything else!"

![20200402_134343_HDR](https://user-images.githubusercontent.com/63064421/78292206-4bbcf500-74ec-11ea-9ee5-f0ecdef259c2.jpg)
![20200402_134252_HDR](https://user-images.githubusercontent.com/63064421/78292218-4f507c00-74ec-11ea-8c81-760b61ced51d.jpg)
![20200402_133550_HDR](https://user-images.githubusercontent.com/63064421/78292227-524b6c80-74ec-11ea-9627-af1cebdb5261.jpg)

**Comment by carnage702** [comment]
> 
> 
> Well I replied via email, but it doesn't look like it made it to the site for some reason so here it is...
> 
> "Thanks so much for the super quick response! I took a picture of my settings and attached it to this email per your request. I also saw in the other post you requested a picture of the OP's USB HDD Root and save folder so I included screenshots of those too. Please let me know if you need anything else!"
> 
> ![20200402_134343_HDR](https://user-images.githubusercontent.com/63064421/78292206-4bbcf500-74ec-11ea-9ee5-f0ecdef259c2.jpg)
> ![20200402_134252_HDR](https://user-images.githubusercontent.com/63064421/78292218-4f507c00-74ec-11ea-8c81-760b61ced51d.jpg)
> ![20200402_133550_HDR](https://user-images.githubusercontent.com/63064421/78292227-524b6c80-74ec-11ea-9627-af1cebdb5261.jpg)

well im guessing your flashdrive isnt fast enought and time outs while saving the memory card so im gonna tell you a way to do this and hopefully it works.

see all the 0kb files? and their names? remenber the names and delete them all.
next get the animal crossing save file(the one that isnt 0kb, and copy it a bunch of times, then rename each copy to each of those 0kb files, that should work.

**Comment by BeenaSon** [comment]
OMG!  You are a genius!  That worked!  I basically just made 16 copies of the only legit save file I had (for Animal Crossing) since I have 17 games currently.  Then, I changed the file names of each of those 16 save files so it matched the first 4 digits of the Game ID tied to the games I have.  I then launched about 5 games that previously failed to launch with memory card emulation enabled and they all worked fine!  I even confirmed that i was able to save my game in each title as well and it worked!  Thanks again and I hope this solution can help someone else with the same issue!
